src/historical.rs

```diff
@@ -113,10 +109,6 @@ impl Historical {
         let db = WalletDB::get_connection();
         let historical = db.collection("historical");
 
-        // We search for historical prices over a week to make sure we get
```

Comment:
Keep the first part of the comment please =)

Reply:
ok!